Makefile

```diff
@@ -55,7 +55,7 @@ export GO111MODULE=on
 # Add the tools bin to the front of the path
 export PATH := $(TOOLSBIN):$(PATH)
 
-all: build
```

Comment:
removing the cmd/ directory in previous PR broke the go-build target. It's otherwise unused, remove it.